build-openssl-curl.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Usage: run this script from the root of the Simplified-iOS repo.
+#
+# Summary: this script rebuilds OpenSSL 1.0.1u and cURL 7.64.1 which are
+#              required by the Adobe RMSDK.
+#
+# In theory, following the instructions in "adobe-rmsdk/RMSDK_User_Manual(obj).pdf",
+# you should be able to build OpenSSL (section 12.1) and cURL (section 12.3)
+# since Adobe provides this package to their developers. The following are some
+# smoother steps to achieve that.
+
+# Note: if you want/need to use an Xcode installed at a location other than
+# /Applications, you'll need to update the $DEVELOPER env variable mentioned
+# at the top of both the build.sh / build_curl.sh scripts below.
+
+cp build_curl.sh adobe-rmsdk/thirdparty/curl/iOS-libcurl/
+SDKVERSION=`xcodebuild -version -sdk iphoneos | grep SDKVersion | sed 's/SDKVersion[: ]*//'`
+
+echo "======================================="
+echo "Building OpenSSL..."
+cd adobe-rmsdk/thirdparty/openssl
+mkdir public
+mv iOS-openssl/ public/ios/
+cd public/ios
+curl -O https://www.openssl.org/source/old/1.0.1/openssl-1.0.1u.tar.gz
```

Comment:
I was reluctant to make these steps into a script for fear that the download URL may change in the future or become outdated.
To make it clear to the developer that this should be configurable in the future, I would extract the target URL for OpenSSL and the OpenSSL version ID into defined variables at the top of the file with a comment saying "edit as required if OpenSSL needs updating or retargeting" and use those variables in the actual downloading and extracting process.

Reply:
good point, will change